tools/server/index.js

```diff
@@ -46,7 +46,7 @@ const devServer = (app, conf) => {
   if (conf.type == "sl1") {
     app.set("printer", new PrinterSL1());
   } else {
-    app.set("printer", new PrinterMK3());
```

Comment:
But there is mini

Reply:
Maybe its better use:
`  if (conf.printerFamily == "sla") {
    app.set("printer", new PrinterSLA());
  } else {
    app.set("printer", new PrinterFDM());
}
`